pose-detection/src/blazepose_tfjs/detector.ts

```diff
@@ -478,9 +543,23 @@ class BlazePoseTfjsDetector implements PoseDetector {
       auxiliaryLandmarksFiltered = this.landmarksSmoothingFilterAuxiliary.apply(
           auxiliaryLandmarksFiltered, this.timestamp, imageSize,
           true /* normalized */, objectScaleROI);
+
+      // Smoothes pose world landmark coordinates to reduce jitter.
+      if (this.worldLandmarksSmoothingFilterActual == null) {
+        this.worldLandmarksSmoothingFilterActual = new KeypointsSmoothingFilter(
+            constants.BLAZEPOSE_WORLD_LANDMARKS_SMOOTHING_CONFIG_ACTUAL);
+      }
+      actualWorldLandmarksFiltered =
+          this.worldLandmarksSmoothingFilterActual.apply(
+              actualWorldLandmarks, this.timestamp, imageSize,
+              true /* normalized */, objectScaleROI);
```

Comment:
Two things here:
- How "normalized = true" affects smoothing? Will it be scaled with image size? It yes, then set to false and pass `null` as imageSize to be safe.
- I know that world landmarks smoothing config says not to scale, but to be safe pass either `null` or `1` as a scale and add comment about it.

Reply:
This change was temporary and is now gone in the newer commits, it is now normalized = false